obu/.gitignore

```diff
@@ -0,0 +1,33 @@
+# Git
```

Comment:
Don't put the `.gitignore` in subdirectories.

Reply:
I am not sure what effect this gitignore will have on other directories, and personally understand that this repo contains individual modules, so maybe it is better to put this gitignore in a subdirectory?

Reply:
It's ugly for every sub directory in a git repo to have its own `.gitignore`.
Update the one at `APOLLO_CONTRIB_ROOT_DIR`.